BeagleBone_black/src/motor_control/main.cpp

```diff
@@ -135,76 +138,76 @@ void Main::accelerateMotors()
+  while (state_.current_state == data::State::kDecelerating) {
     // Check for state machine critical failure flag
-    state = data.getStateMachineData();
-    if (state.critical_failure) {
+    state_ = data_.getStateMachineData();
+    if (state_.critical_failure) {
       this->stopMotors();
       goto exit_loop;
     }
 
     // Check for motors critical failure flag
-    motorFailure = motor->checkStatus();
-    if (motorFailure) {
+    motorFailure_ = motor_.checkStatus();
+    if (motorFailure_) {
       log_.INFO("MOTOR", "Motor State: Motor Failure\n");
-      MotorsRpm motors_rpm = motor->getSpeed();
-      motor_data = {
+      MotorsRpm motors_rpm = motor_.getSpeed();
+      motor_data_ = {
         data::MotorState::kCriticalFailure,
-        motors_rpm.rpm_FL,
-        motors_rpm.rpm_FR,
-        motors_rpm.rpm_BL,
-        motors_rpm.rpm_BR };
-      data.setMotorData(motor_data);
+        motors_rpm.rpm_FL_,
+        motors_rpm.rpm_FR_,
+        motors_rpm.rpm_BL_,
+        motors_rpm.rpm_BR_ };
+      data_.setMotorData(motor_data_);
       this->stopMotors();
     }
 
     // Step down motor RPM
     log_.INFO("MOTOR", "Motor State: Decelerating\n");
-    nav = data.getNavigationData();
-    rpm = calculateDecelerationRPM(nav.velocity);
-    motor->setSpeed(rpm);
-    MotorsRpm motors_rpm = motor->getSpeed();
+    nav_ = data_.getNavigationData();
+    rpm_ = calculateDecelerationRPM(nav_.velocity);
+    motor_.setSpeed(rpm_);
+    MotorsRpm motors_rpm = motor_.getSpeed();
     // Updates the shared data on the motors RPM
-    motor_data = {
+    motor_data_ = {
       data::MotorState::kMotorDecelerating,
-      motors_rpm.rpm_FL,
-      motors_rpm.rpm_FR,
-      motors_rpm.rpm_BL,
-      motors_rpm.rpm_BR };
-    data.setMotorData(motor_data);
+      motors_rpm.rpm_FL_,
+      motors_rpm.rpm_FR_,
+      motors_rpm.rpm_BL_,
+      motors_rpm.rpm_BR_ };
+    data_.setMotorData(motor_data_);
   }
   exit_loop: ;
 }
 
 void Main::stopMotors()
 {
-  motor->setSpeed(0);
+  motor_.setSpeed(0);
   bool allMotorsStopped = false;
   // Updates the shared data on the motors RPM while the motor is trying to stop
   while (!allMotorsStopped) {
     log_.DBG2("MOTOR", "Motor State: Stopping\n");
-    MotorsRpm motors_rpm = motor->getSpeed();
-    data::Motors motor_data = {
+    MotorsRpm motors_rpm = motor_.getSpeed();
+    data::Motors motor_data_ = {
       data::MotorState::kMotorStopping,
-      motors_rpm.rpm_FL,
-      motors_rpm.rpm_FR,
-      motors_rpm.rpm_BL,
-      motors_rpm.rpm_BR };
-    data.setMotorData(motor_data);
-    if (motors_rpm.rpm_FL == 0 && motors_rpm.rpm_FR == 0 &&
-      motors_rpm.rpm_BL == 0 && motors_rpm.rpm_BR == 0)
+      motors_rpm.rpm_FL_,
+      motors_rpm.rpm_FR_,
+      motors_rpm.rpm_BL_,
+      motors_rpm.rpm_BR_ };
+    data_.setMotorData(motor_data_);
+    if (motors_rpm.rpm_FL_ == 0 && motors_rpm.rpm_FR_ == 0 &&
+      motors_rpm.rpm_BL_ == 0 && motors_rpm.rpm_BR_ == 0)
```

Comment:
This has to be a 4 space indent. Check the rest of your code for this mistake as well.